tests/Rector/ClassMethod/CommandConstantReturnCodeRector/Fixture/skip_other_function_call_return_int.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\CommandConstantReturnCodeRector\Fixture;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SkipAlreadyUsingConstantNestedFunction extends Command
```

Comment:
this fixture can be removed, out of rule context. 

New Rector rule should be used to verify if private method called in `execute()`

Reply:
++